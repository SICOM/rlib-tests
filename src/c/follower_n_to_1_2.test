COMMAND=./follower_n_to_1_2
FORMATS="pdf xml txt csv html"
EXPECTED_pdf=
EXPECTED_xml=
EXPECTED_txt=
EXPECTED_csv=
EXPECTED_html=
