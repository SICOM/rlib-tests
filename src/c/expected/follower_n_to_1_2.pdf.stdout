%PDF-1.3
%·¾­ª
1 0 obj
<<
/Type /Catalog
/Pages 2 0 R
>>
endobj
2 0 obj
<<
/Type /Pages
/Kids [ 4 0 R ]
/Count 1
>>
endobj
3 0 obj
<<
/Producer (Haru Free PDF Library 2.2.1)
/Title (RLIB Report)
>>
endobj
4 0 obj
<<
/Type /Page
/MediaBox [ 0 0 612 792 ]
/Contents 5 0 R
/Resources <<
/ProcSet [ /PDF /Text /ImageB /ImageC /ImageI ]
/Font <<
/F1 7 0 R
/F2 8 0 R
/F3 9 0 R
/F4 10 0 R
>>
>>
/Parent 2 0 R
/Rotate 270
/Annots [ 11 0 R 12 0 R 13 0 R 14 0 R 15 0 R 16 0 R 17 0 R 18 0 R 19 0 R 20 0 R 21 0 R 22 0 R ]
>>
endobj
5 0 obj
<<
/Length 6 0 R
>>
stream
1 0 0 1 0 792 cm
0 -1 1 0 0 0 cm
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
BT
1 0 0 1 14.4 16.56 Tm
(Page:   1) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 11 Tf
BT
1 0 0 1 14.4 589.23999 Tm
(Test Report \050Normal\051 AND THE REPORT FORMAT IS: PDF       3    ) Tj
ET
/F1 9 Tf
/F1 11 Tf
/F2 11 Tf
BT
1 0 0 1 14.4 578.23999 Tm
(Test Report \050Bold\051) Tj
ET
/F1 11 Tf
/F1 9 Tf
/F1 11 Tf
/F3 11 Tf
BT
1 0 0 1 14.4 567.23999 Tm
(Test Report \050Italics\051) Tj
ET
/F1 11 Tf
/F1 9 Tf
/F1 11 Tf
/F2 11 Tf
/F4 11 Tf
BT
1 0 0 1 14.4 556.23999 Tm
(Test Report \050Bold and Italics\051) Tj
ET
/F3 11 Tf
/F1 11 Tf
/F1 9 Tf
/F1 11 Tf
/F2 11 Tf
/F4 11 Tf
BT
1 0 0 1 14.4 545.23999 Tm
(BOO: ) Tj
ET
/F3 11 Tf
/F1 11 Tf
/F1 9 Tf
/F1 11 Tf
BT
1 0 0 1 47.4 545.23999 Tm
(mike is a ) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
0 0 0 rg
0 0 0 RG
14.4 537.59998 763.20001 0.856 re
f
0.89804 0.89804 0.89804 rg
0.89804 0.89804 0.89804 RG
14.4 528.59998 334.79999 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 14.4 530.76001 Tm
(First Name          ) Tj
ET
BT
1 0 0 1 122.4 530.76001 Tm
( Last Name           ) Tj
ET
BT
1 0 0 1 235.8 530.76001 Tm
( ) Tj
ET
BT
1 0 0 1 241.2 530.76001 Tm
(Color               ) Tj
ET
14.4 527.59998 763.20001 0.856 re
f
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
1 1 0 rg
1 1 0 RG
14.4 514.59998 108 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 14.4 516.76001 Tm
(Bob                 ) Tj
ET
BT
1 0 0 1 122.4 516.76001 Tm
( ) Tj
ET
1 1 0 rg
1 1 0 RG
127.8 514.59998 108 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 127.8 516.76001 Tm
(Doan                ) Tj
ET
BT
1 0 0 1 235.8 516.76001 Tm
( ) Tj
ET
BT
1 0 0 1 241.2 516.76001 Tm
(blue                       NO_MATCH NO_MATCH NO_MATCH) Tj
ET
BT
1 0 0 1 527.40002 516.76001 Tm
( ) Tj
ET
BT
1 0 0 1 532.79999 516.76001 Tm
(NO_MATCH) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
1 1 0 rg
1 1 0 RG
14.4 505.60001 108 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 14.4 507.76001 Tm
(Terry               ) Tj
ET
0.89804 0.89804 0.89804 rg
0.89804 0.89804 0.89804 RG
122.4 505.60001 5.4 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 122.4 507.76001 Tm
( ) Tj
ET
1 1 0 rg
1 1 0 RG
127.8 505.60001 108 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 127.8 507.76001 Tm
(Doan                ) Tj
ET
0.89804 0.89804 0.89804 rg
0.89804 0.89804 0.89804 RG
235.8 505.60001 340.20001 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 235.8 507.76001 Tm
( ) Tj
ET
BT
1 0 0 1 241.2 507.76001 Tm
(blue                       NO_MATCH NO_MATCH NO_MATCH) Tj
ET
BT
1 0 0 1 527.40002 507.76001 Tm
( ) Tj
ET
BT
1 0 0 1 532.79999 507.76001 Tm
(NO_MATCH) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
1 1 0 rg
1 1 0 RG
14.4 496.60001 108 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 14.4 498.76001 Tm
(Eric                ) Tj
ET
BT
1 0 0 1 122.4 498.76001 Tm
( ) Tj
ET
1 1 0 rg
1 1 0 RG
127.8 496.60001 108 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 127.8 498.76001 Tm
(Buruschkin          ) Tj
ET
BT
1 0 0 1 235.8 498.76001 Tm
( ) Tj
ET
BT
1 0 0 1 241.2 498.76001 Tm
(green                      NO_MATCH NO_MATCH NO_MATCH) Tj
ET
BT
1 0 0 1 527.40002 498.76001 Tm
( ) Tj
ET
BT
1 0 0 1 532.79999 498.76001 Tm
(NO_MATCH) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
1 1 0 rg
1 1 0 RG
14.4 487.60001 108 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 14.4 489.76001 Tm
(Mike                ) Tj
ET
0.89804 0.89804 0.89804 rg
0.89804 0.89804 0.89804 RG
122.4 487.60001 5.4 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 122.4 489.76001 Tm
( ) Tj
ET
1 1 0 rg
1 1 0 RG
127.8 487.60001 108 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 127.8 489.76001 Tm
(Roth                ) Tj
ET
0.89804 0.89804 0.89804 rg
0.89804 0.89804 0.89804 RG
235.8 487.60001 340.20001 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 235.8 489.76001 Tm
( ) Tj
ET
BT
1 0 0 1 241.2 489.76001 Tm
(yellow                     NO_MATCH NO_MATCH NO_MATCH) Tj
ET
BT
1 0 0 1 527.40002 489.76001 Tm
( ) Tj
ET
BT
1 0 0 1 532.79999 489.76001 Tm
(NO_MATCH) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
1 1 0 rg
1 1 0 RG
14.4 478.60001 108 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 14.4 480.76001 Tm
(Bob                 ) Tj
ET
BT
1 0 0 1 122.4 480.76001 Tm
( ) Tj
ET
1 1 0 rg
1 1 0 RG
127.8 478.60001 108 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 127.8 480.76001 Tm
(Kratz               ) Tj
ET
BT
1 0 0 1 235.8 480.76001 Tm
( ) Tj
ET
BT
1 0 0 1 241.2 480.76001 Tm
(pink                       NO_MATCH NO_MATCH NO_MATCH) Tj
ET
BT
1 0 0 1 527.40002 480.76001 Tm
( ) Tj
ET
BT
1 0 0 1 532.79999 480.76001 Tm
(NO_MATCH) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
1 1 0 rg
1 1 0 RG
14.4 469.60001 108 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 14.4 471.76001 Tm
(Steve               ) Tj
ET
0.89804 0.89804 0.89804 rg
0.89804 0.89804 0.89804 RG
122.4 469.60001 5.4 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 122.4 471.76001 Tm
( ) Tj
ET
1 1 0 rg
1 1 0 RG
127.8 469.60001 108 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 127.8 471.76001 Tm
(Tilden              ) Tj
ET
0.89804 0.89804 0.89804 rg
0.89804 0.89804 0.89804 RG
235.8 469.60001 340.20001 8.856 re
f
0 0 0 rg
0 0 0 RG
BT
1 0 0 1 235.8 471.76001 Tm
( ) Tj
ET
BT
1 0 0 1 241.2 471.76001 Tm
(purple                     NO_MATCH NO_MATCH NO_MATCH) Tj
ET
BT
1 0 0 1 527.40002 471.76001 Tm
( ) Tj
ET
BT
1 0 0 1 532.79999 471.76001 Tm
(NO_MATCH) Tj
ET
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf
/F1 9 Tf

endstream
endobj
6 0 obj
5365
endobj
7 0 obj
<<
/Type /Font
/BaseFont /Courier
/Subtype /Type1
/Encoding /WinAnsiEncoding
>>
endobj
8 0 obj
<<
/Type /Font
/BaseFont /Courier-Bold
/Subtype /Type1
/Encoding /WinAnsiEncoding
>>
endobj
9 0 obj
<<
/Type /Font
/BaseFont /Courier-Oblique
/Subtype /Type1
/Encoding /WinAnsiEncoding
>>
endobj
10 0 obj
<<
/Type /Font
/BaseFont /Courier-BoldOblique
/Subtype /Type1
/Encoding /WinAnsiEncoding
>>
endobj
11 0 obj
<<
/Rect [ 514.59998 669.59998 523.59998 777.59998 ]
/Type /Annot
/Subtype /Link
/A <<
/Type /Action
/S /URI
/URI (http:\057\057www.yahoo.com)
>>
/P 4 0 R
>>
endobj
12 0 obj
<<
/Rect [ 514.59998 556.20001 523.59998 664.20001 ]
/Type /Annot
/Subtype /Link
/A <<
/Type /Action
/S /URI
/URI (http:\057\057www.yahoo.com)
>>
/P 4 0 R
>>
endobj
13 0 obj
<<
/Rect [ 505.60001 669.59998 514.59998 777.59998 ]
/Type /Annot
/Subtype /Link
/A <<
/Type /Action
/S /URI
/URI (http:\057\057www.yahoo.com)
>>
/P 4 0 R
>>
endobj
14 0 obj
<<
/Rect [ 505.60001 556.20001 514.59998 664.20001 ]
/Type /Annot
/Subtype /Link
/A <<
/Type /Action
/S /URI
/URI (http:\057\057www.yahoo.com)
>>
/P 4 0 R
>>
endobj
15 0 obj
<<
/Rect [ 496.60001 669.59998 505.60001 777.59998 ]
/Type /Annot
/Subtype /Link
/A <<
/Type /Action
/S /URI
/URI (http:\057\057www.yahoo.com)
>>
/P 4 0 R
>>
endobj
16 0 obj
<<
/Rect [ 496.60001 556.20001 505.60001 664.20001 ]
/Type /Annot
/Subtype /Link
/A <<
/Type /Action
/S /URI
/URI (http:\057\057www.yahoo.com)
>>
/P 4 0 R
>>
endobj
17 0 obj
<<
/Rect [ 487.60001 669.59998 496.60001 777.59998 ]
/Type /Annot
/Subtype /Link
/A <<
/Type /Action
/S /URI
/URI (http:\057\057www.yahoo.com)
>>
/P 4 0 R
>>
endobj
18 0 obj
<<
/Rect [ 487.60001 556.20001 496.60001 664.20001 ]
/Type /Annot
/Subtype /Link
/A <<
/Type /Action
/S /URI
/URI (http:\057\057www.yahoo.com)
>>
/P 4 0 R
>>
endobj
19 0 obj
<<
/Rect [ 478.60001 669.59998 487.60001 777.59998 ]
/Type /Annot
/Subtype /Link
/A <<
/Type /Action
/S /URI
/URI (http:\057\057www.yahoo.com)
>>
/P 4 0 R
>>
endobj
20 0 obj
<<
/Rect [ 478.60001 556.20001 487.60001 664.20001 ]
/Type /Annot
/Subtype /Link
/A <<
/Type /Action
/S /URI
/URI (http:\057\057www.yahoo.com)
>>
/P 4 0 R
>>
endobj
21 0 obj
<<
/Rect [ 469.60001 669.59998 478.60001 777.59998 ]
/Type /Annot
/Subtype /Link
/A <<
/Type /Action
/S /URI
/URI (http:\057\057www.yahoo.com)
>>
/P 4 0 R
>>
endobj
22 0 obj
<<
/Rect [ 469.60001 556.20001 478.60001 664.20001 ]
/Type /Annot
/Subtype /Link
/A <<
/Type /Action
/S /URI
/URI (http:\057\057www.yahoo.com)
>>
/P 4 0 R
>>
endobj
xref
0 23
0000000000 65535 f
0000000015 00000 n
0000000064 00000 n
0000000123 00000 n
0000000205 00000 n
0000000520 00000 n
0000005939 00000 n
0000005959 00000 n
0000006054 00000 n
0000006154 00000 n
0000006257 00000 n
0000006365 00000 n
0000006539 00000 n
0000006713 00000 n
0000006887 00000 n
0000007061 00000 n
0000007235 00000 n
0000007409 00000 n
0000007583 00000 n
0000007757 00000 n
0000007931 00000 n
0000008105 00000 n
0000008279 00000 n
trailer
<<
/Root 1 0 R
/Info 3 0 R
/Size 23
>>
startxref
8453
%%EOF
